airflow/operators/datetime_branch_operator.py

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
the suffix of `_operator` in the file name should be removed (to comply with AIP 21)
see https://github.com/apache/airflow/issues/11178